sky/provision/vast/utils.py

```diff
@@ -0,0 +1,103 @@
+# pylint: disable=assignment-from-no-return
```

Comment:
Why do we need this? Can we remove it?

Reply:
the sdk is actually weird. we made a CLI first and currently it's a fairly unusual [wrapper around the cli](https://github.com/kristopolous/vastai-sdk/blob/main/vastai_sdk/vastai_sdk.py) ... unusual enough to confuse pylint.  I may internally refactor the sdk to be a more conventional structure eventually but I'd need to convince a bunch of people that it's worthy of time.

Alternatively, I can do a deep dive into pylint and see what's causing this false positive and modify the sdk to make it happy.

Reply:
SG! Please left an comment for this (and maybe a todo) ;)

Reply:
see 5f9cc3bb9f4ec9a274d93988bd0192d6a8283b92